src/scanner/mod.rs

```diff
@@ -21,12 +21,13 @@ use std::{
 /// Timeout is the time RustScan should wait before declaring a port closed. As datatype Duration.
 /// Quiet is whether or not RustScan should print things, or wait until the end to print only open ports.
 #[cfg(not(tarpaulin_include))]
+#[derive(Debug)]
 pub struct Scanner {
-    ips: Vec<IpAddr>,
+    pub ips: Vec<IpAddr>,
```

Comment:
Why are these made public? :)

Reply:
I made them public so we can calculate average socket connection time based on number of ips, ports and runtime.